run_comparison.py

```diff
@@ -0,0 +1,75 @@
+import copy
+from copy import deepcopy
+from run_sim import run
+from config_scripts.parse_args import parse_args
+from config_scripts.setup_config import read_config
+import json
+import sys
+import os
+import configparser
+
+args_obj = parse_args()
+
+def run_comparison():
+    all_sims_dict = read_config(args_obj=args_obj, config_path=args_obj['config_path'])
+
+    # Call the run function from run_sim.py
+    run(sims_dict=all_sims_dict)
+
+def find_type_and_saved_path(config_path=args_obj['config_path']):
+    config = configparser.ConfigParser()
+
+    #Following is from setup_config to reach the type of path
+    if config_path is None:
+        config_path = os.path.join('ini', 'run_ini', 'config.ini')
+    config.read(config_path)
+
+    #Following is from setup_config to reach the type of path
+    if not config.has_option('general_settings', 'sim_type'):
+        config_path = os.path.join('ini', 'run_ini')
+        create_dir(config_path)
+        raise ValueError("Missing 'general_settings' section in the configuration file. "
+                         "Please ensure you have a file called config.ini in the run_ini directory.")
+
+    if config['general_settings']['sim_type'] == 'yue':
+        return './data/run_comparison_data/yue_run_data.json'
+    else:
+        raise ValueError("Error: sim_type not supported by function.")
+
+def compare_json_files(old_file, new_file):
+    ##Load and compare two JSON files.
+    with open(old_file, 'r') as f:
+        old_data = json.load(f)
+
+    with open(new_file, 'r') as g:
+        new_data = json.load(g)
+
+    if old_data == new_data:
```

Comment:
You'll need to loop over one file and compare it with another. Here is where you can ignore things like simulation time, arrival rate, etc.

They are JSON objects which we can use as dictionaries in python, the (rough) logic will look like this:

```
for key, val in old_data.items():
     if old_data[key] != new_data[key]:
          raise ValueError('Some comment.')
```
It won't be exactly like this because you'll have to deal with some dictionary values. I'll leave it to you to try and figure this out and if you get stuck please push to this PR again with the progress and we can help point you in the right direction.